.github/workflows/pr-commit.yaml

```diff
@@ -29,22 +29,30 @@ jobs:
           version: 6.0.2
       - name: Install modules
         run: pnpm i
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install Poetry
+        run: curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python -
+      - name: Setup api-evidence
+        uses: ./.github/actions/setup-python-app
+        with:
+          app-name: 'api-evidence'
+          create-db: false
+      - name: Setup api-dispute
+        uses: ./.github/actions/setup-python-app
+        with:
+          app-name: 'api-dispute'
+          create-db: false
+      - name: Check Python code format
+        run: pnpx nx run-many --target=format --check --all --parallel
```

Comment:
@kelvin-wong Actually, now that I look at it, I think if we're running all the format checks here, the next command isn't needed. Anything from `pnpx nx format:check --base=origin/main` will already be included with your `--all` command.

We could either remove the next command, or change your command to specifically run Python apps. Maybe something for later, since we're probably already spending too much time on CI/devops

Reply:
`pnpx nx format:check --base=origin/main` is about format files (js, json, etc.) using `prettier` which is a built-in command. But this command is about calling the `format` command on python apps.

Reply:
Ahhh, you're right. I was a little confused, maybe we can call the target "format_python"

Not a big deal for now though, nice work 👌

Reply:
maybe later we only keep this command instead of `format:check` if we need more sophisticated formatting requirements for js/ts apps.